flaml/version.py

```diff
@@ -1 +1 @@
-__version__ = "1.2.1"
+__version__ = "1.2.2"
```

Comment:
Do we need to update the version for this change?

Reply:
Yes, to distinguish from the released version. It doesn't mean we'll release right after merging.

Reply:
With the new changes, I'd like to release a new version after merging.